db/migrate/20190922124029_add_misc_to_doc_auth_logs.rb

```diff
@@ -0,0 +1,10 @@
+class AddMiscToDocAuthLogs < ActiveRecord::Migration[5.1]
+  def change
+    safety_assured do
```

Comment:
table was just created and sparsely populated.  we don't need strong migrations here